### PR TITLE
fix bug switching language to device language

### PR DESCRIPTION
**Pull request by EvGen94**
fix bug switching language to device language

### CODE REVIEW

File: lib/pages/select_language.dart [hunk 1]
```diff
@@ -122,24 +122,24 @@ class _SelectLanguageState extends State<SelectLanguage> {
...
-        language = AppLocalizations.of(context)!.spanish;
-        break;
-      case 'ru':
-        language = AppLocalizations.of(context)!.russian;
-        break;
-      default:
-        language = 'English';
-    }
-    return language;
+  switch (languageCode) {
```
Comment (huynguyennovem) [inline lib/pages/select_language.dart:131]: You can have a shorter LOC (line of code) without using more `language` variable:

```dart
switch (languageCode) {
      case 'en':
        return AppLocalizations.of(context)!.english;
        break;
      ...
      default:
        return 'English';
    }
```

**Comment by EvGen94** [comment]
https://user-images.githubusercontent.com/51719953/141752714-2c154bc0-90fd-4ec4-8402-381298339252.mp4

File: lib/repositories/workspaces_repository.dart [hunk 2]
```diff
@@ -46,7 +46,11 @@ class WorkspacesRepository {
                 transform: true,
               ))
           .toList();
-
+      final dataL = await _storage.select(
```
Comment (huynguyennovem) [inline lib/repositories/workspaces_repository.dart:49]: What is the purpose of this? Why do you put a language query here?

File: lib/repositories/workspaces_repository.dart [hunk 3]
```diff
@@ -57,7 +61,11 @@ class WorkspacesRepository {
               workspaceId: workspaceId!,
             ),
           ));
-
+      _storage.update(
```
Comment (huynguyennovem) [inline lib/repositories/workspaces_repository.dart:64]: What is the purpose of this?

**Pull request closed without merging**